atm.js

```diff
@@ -1,21 +1,110 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready( function() {
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//change css property of div.balance to #FF003E
+$("div.balance").css({
+    "background-color": "#FF003E",
+    "border-radius": "10px"
+    });
```

Comment:
If you toggle a class here instead of setting inline styles, you can separate your concerns by keeping behavior in JS files and styles in CSS files.